app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -254,4 +256,69 @@ class SmokeTest {
             }
         }
     }
+
+    @Test
+    fun verifySearchEngineCanBeChangedTemporarilyUsingShortcuts() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        homeScreen {
+        }.openSearch {
+            verifyKeyboardVisibility()
+            clickSearchEngineButton()
+            verifySearchEngineList()
+            verifySearchEngineIconsList()
+            changeDefaultSearchEngine("Amazon.com")
+        }.goToSearchEngine {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+        }.openTabDrawer {
+
+            // Changing search engine to Bing
+        }.openHomeScreen {
+        }.openSearch {
+            clickSearchEngineButton()
+            mDevice.waitForIdle()
+            changeDefaultSearchEngine("Bing")
+        }.goToSearchEngine {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+        }.openTabDrawer {
+
+            // Changing search engine to DuckDuckGo
+        }.openHomeScreen {
+        }.openSearch {
+            clickSearchEngineButton()
+            mDevice.waitForIdle()
+            changeDefaultSearchEngine("DuckDuckGo")
+        }.goToSearchEngine {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+        }.openTabDrawer {
+
+            // Changing search engine to Twitter
+        }.openHomeScreen {
+        }.openSearch {
+            clickSearchEngineButton()
+            mDevice.waitForIdle()
+            changeDefaultSearchEngine("Twitter")
+        }.goToSearchEngine {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+        }.openTabDrawer {
+
+            // Changing search engine to Wikipedia
+        }.openHomeScreen {
+        }.openSearch {
+            clickSearchEngineButton()
+            scrollToElementByText("Wikipedia")
+            changeDefaultSearchEngine("Wikipedia")
+        }.goToSearchEngine {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+        }.openTabDrawer {
+        }.openHomeScreen {

```

Comment:
I will add a comment here, Changing to default search engine, or something similar